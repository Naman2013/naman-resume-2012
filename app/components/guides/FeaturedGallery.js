import React from 'react';
import PropTypes from 'prop-types';
import style from './FeaturedGallery.style';
import SterlingTitle from 'components/common/titles/SterlingTitle';

const FeaturedGallery = () => (
  <div className="root">
    <SterlingTitle
      title="Featured observation"
      subTitle="Community observation"
      theme={{ title: { color: 'white' }, subTitle: { color: 'white' } }}
    />
    <style jsx>{style}</style>
  </div>
);

export default FeaturedGallery;
