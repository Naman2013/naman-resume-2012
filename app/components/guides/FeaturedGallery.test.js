import React from 'react';
import { shallow } from 'enzyme';
import FeaturedGallery from './FeaturedGallery';

describe('FeaturedGallery', () => {
  const props = {
    items: [],
  };
  const shallowWrapper = shallow(<FeaturedGallery {...props} />);
  it('should render correctly', () => {
    expect(shallowWrapper).toMatchSnapshot();
  });

  it('should render a `SterlingTitle`', () => {
    expect(shallowWrapper.find('SterlingTitle').length).toEqual(1);
  });
});
