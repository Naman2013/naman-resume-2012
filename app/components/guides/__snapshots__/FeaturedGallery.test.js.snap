// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedGallery should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FeaturedGallery
    items={Array []}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SterlingTitle
          subTitle="Community observation"
          theme={
            Object {
              "subTitle": Object {
                "color": "white",
              },
              "title": Object {
                "color": "white",
              },
            }
          }
          title="Featured observation"
        />,
        <JSXStyle
          css={
            Array [
              ".root.jsx-3854175346{background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);}",
            ]
          }
          styleId="3854175346"
        />,
      ],
      "className": "jsx-3854175346 root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "subTitle": "Community observation",
          "theme": Object {
            "subTitle": Object {
              "color": "white",
            },
            "title": Object {
              "color": "white",
            },
          },
          "title": "Featured observation",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "css": Array [
            ".root.jsx-3854175346{background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);}",
          ],
          "styleId": "3854175346",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SterlingTitle
            subTitle="Community observation"
            theme={
              Object {
                "subTitle": Object {
                  "color": "white",
                },
                "title": Object {
                  "color": "white",
                },
              }
            }
            title="Featured observation"
          />,
          <JSXStyle
            css={
              Array [
                ".root.jsx-3854175346{background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);}",
              ]
            }
            styleId="3854175346"
          />,
        ],
        "className": "jsx-3854175346 root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "subTitle": "Community observation",
            "theme": Object {
              "subTitle": Object {
                "color": "white",
              },
              "title": Object {
                "color": "white",
              },
            },
            "title": "Featured observation",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "css": Array [
              ".root.jsx-3854175346{background:url(https://vega.slooh.com/assets/v4/common/Dark_Pattern_Featured_LargeCanvas.png);}",
            ],
            "styleId": "3854175346",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
